doc/bot.rst

```diff
@@ -0,0 +1,29 @@
+Autocrypt bot
++++++++++++++++
+
+Implemented using `py-autocrypt`_.
```

Comment:
now that this is a different page from dev-status.rst, this link doesn't work any more. :(